app/Http/Controllers/TorrentController.php

```diff
@@ -861,6 +861,13 @@ public function torrent($slug, $id)
                 $movie = $client->scrape('movie', 'tt'.$torrent->imdb);
             }
         }
+        
+        if ($movie->recommendations){
+            $movie->recommendations['results'] = array_map(function($recomentaion){
+                $recomentaion['exists'] = Torrent::where("tmdb", $recomentaion['id'])->get()->isNotEmpty();
```

Comment:
Small and petty I know but UNIT3D standard is to define the operator. 
`Torrent::where("tmdb", $recomentaion['id'])->get()->isNotEmpty();`
Instead
`Torrent::where('tmdb', '=', $recomentaion['id'])->get()->isNotEmpty();`